frontend/src/views/Combat.vue

```diff
@@ -286,10 +290,24 @@ export default {
         this.error = e.message;
       }
     },
+
     formattedSkill(skill) {
       const skillBalance = Web3.utils.fromWei(skill, 'ether');
       return `${new BN(skillBalance).toFixed(6)} SKILL`;
     },
+
+    onChooseWeapon(targetToFight) {
```

Comment:
this function is confusingly named

Reply:
right, yeah I did thename when the function was still an  event handler for an emit... it should be  changed.